packages/web/components/Setup/SetupMemberships.tsx

```diff
@@ -78,3 +81,43 @@ export const SetupMemberships: React.FC<SetupMembershipsProps> = ({
     </FlexContainer>
   );
 };
+
+type MembershipListingProps = {
+  member: Membership;
+};
+
+const MembershipListing: React.FC<MembershipListingProps> = ({ member }) => {
```

Comment:
Let's pull this shared component up into its own file, along with the `getDaoLink`, `getImageMoloch` etc functions, so that both  `SetupMemberships` and `PlayerMemberships` can use it

Reply:
okay makes sense